src/PKSim.Infrastructure/InfrastructureRegister.cs

```diff
@@ -50,6 +48,10 @@ public class InfrastructureRegister : Register
    {
       public static IContainer Initialize(bool registerContainerAsStatic = true)
       {
+         Thread.CurrentThread.CurrentCulture = new CultureInfo("en-US");
```

Comment:
@Yuri05 I moved this to the code that is called also for TESTS
Hopefully this will fix your test issues